src/main.cpp

```diff
@@ -70,20 +73,19 @@ int main(const int argc, char** argv) {
         [](double delta_time) { world.phys_tick(delta_time); },
 
         // Graphics
-        [&gbuffers, &ticker](double delta_time, double subtick_time) {
+        [&gbuffers, &ticker, &fps_display, &tps_display](double delta_time,
```

Comment:
Maybe we should just catch all by reference? I don't think it's very important to prohibit the use of other variables in this lambda expression. Moreover, since this lambda is inside the main function, the user can see which variables are being used. It would be a lot of code if we wanted to use many different prefixes.